src/components/AsideHeader/components/AllPagesPanel/AllPagesPanel.scss

```diff
@@ -5,13 +5,12 @@
     height: 100%;
     padding: var(--g-spacing-4) var(--g-spacing-6);
     box-sizing: border-box;
+    --gn-aside-header-item-expanded-radius: 6px;
 
     &__content {
         overflow: auto;
         flex: 1;
 
-        margin: 0 calc(var(--g-spacing-6) * -1);
-
         &_edit-mode {
```

Comment:
Uses _edit-mode with hyphen but the component uses editMode (camelCase) in b('item', {editMode: true}).